apps/mobile/src/navigations/questions-stack/questions-stack.tsx

```diff
@@ -0,0 +1,68 @@
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import React from "react";
+
+import { NumericalValue, QuestionsStackName } from "~/libs/enums/enums";
+import { useAppSelector } from "~/libs/hooks/hooks";
+import { type QuestionsStackNavigationParameterList } from "~/libs/types/types";
+import { BottomTabsNavigator } from "~/navigations/bottom-tabs-navigator/bottom-tabs-navigator";
+import { NotificationQuestions } from "~/screens/notification-questions/notification-questions";
+import { Onboarding } from "~/screens/onboarding/onboarding";
+import { Quiz } from "~/screens/quiz/quiz";
+import { QuizEntry } from "~/screens/quiz-entry/quiz-entry";
+import { Welcome } from "~/screens/welcome/welcome";
```

Comment:
will it break the project if you create a screens/screens.ts file that exports all of the screens from the nested folders so that you can get all of the screen inside of this file using a single import statement?

Reply:
Creating a `screens.ts` file that exports all of the screens won’t break the project. However, since the screens are only used once in the navigation setup, there may not be significant benefits to creating an additional file. It can simplify imports if we are reusing screens across multiple places, but in this case, the existing structure already works well, and adding an extra layer might not bring much efficiency.